### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShurupJosephson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # lab-repo-testing
 Репозиторий для тестирования Labaratory-Repo-Testing
 
-ShurupJosephson's commit
+Another commit
```
Comment (AlexandrOsiptsov) [inline README.md:4]: Change this

**Review by AlexandrOsiptsov — CHANGES_REQUESTED** [review]
Check all _comments_ and make _changes_. **Waiting for you!**

File: README.md (hunk 1 above)
Comment (ShurupJosephson) [reply README.md:4]: Yes, sir! Gonna be done immediately!

Comment (ShurupJosephson) [reply README.md]: This commit fixed all!
fcd3875bd8a0fc38a4979f604769376d2c583527

**Review by AlexandrOsiptsov — APPROVED** [review]
All right, now, that's a deal!

**Pull request closed without merging**